design-system/uilib/src/components/dpdhl-toast/dpdhl-toast.stories.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+	import { Meta, Template, Story } from "@storybook/addon-svelte-csf";
+	import "@sprinteins/dpdhl-uilib";
+	import { Type } from "./type";
+
+	let open = true;
+
+	function handleClose() {
+		open = false;
+	}
+
+	function handleOpen() {
+		open = true;
+	}
+
+</script>
+
+<!-- https://www.dpdhl-brands.com/dpdhl-group/en/guides/touchpoints/user-interface/components/notifications.html -->
+<Template let:args>
+	<dpdhl-toast
+		{open}
+		on:closeToast={handleClose}
+		type="error"
+		title="Toast Title"
+		message="And here we go with a more detailed message."
+		{...args}
```

Comment:
in the story it would be good if could write out all the arguments, because this an example and it would be easier to understand how the component works